homework-0037/src/App.tsx

```diff
@@ -0,0 +1,42 @@
+import ButtonComponent from './components/ButtonComponent';
+import Input from './components/Input';
+import Todo from './components/Todo';
+import React from 'react';
+type todoQuestt = [{quest:string,id:number,key:number
+
+}];
+
+
+function App() {
+  let todoQuestt = [
+    { quest: 'Woshez', id: 0, key: 0 },
+    { quest: 'swiming', id: 1, key: 1 },
+    { quest: 'jumping', id: 2, key: 2 }
+  ];
+
+  const [todoQuest, setCount] = React.useState(todoQuestt);
```

Comment:
```suggestion
  const [todoQuest, setTodoQuest] = React.useState(todoQuestt);
```
такий неймінг буде відповідати конвенції